### DISCUSSION TITLE
Annotation Guideline for NER using Spacy

### DISCUSSION THREAD

**Discussion by karndeepsingh** [explosion/spaCy#9436] [Help: Coding & Implementations] [answered]
Hi,
I have used Label-studio for NER annotation and got the output in below format. Please help me to convert it into Spacy format and provide some understanding how I can do some changes to get into spacy format.

-DOCSTART- -X- O
Paris -X- _ B-PARTY ROLE
is -X- _ O
in -X- _ O
France -X- _ B-PARTY ADDRESS

London -X- _ B-PARTY NAME
is -X- _ O
in -X- _ O
UK -X- _ B-PARTY ADDRESS

BORROWER -X- _ B-PARTY ROLE
Synergy -X- _ B-PARTY NAME
One -X- _ I-PARTY NAME
Lending, -X- _ I-PARTY NAME
Inc. -X- _ I-PARTY NAME
dba -X- _ I-PARTY NAME
Mutual -X- _ I-PARTY NAME
of -X- _ I-PARTY NAME
Omaha -X- _ I-PARTY NAME
Mortgage -X- _ I-PARTY NAME
5716 -X- _ B-PARTY ADDRESS
Corsa -X- _ I-PARTY ADDRESS
Avenuew, -X- _ I-PARTY ADDRESS
Sulle -X- _ I-PARTY ADDRESS
102 -X- _ I-PARTY ADDRESS
Westlake -X- _ I-PARTY ADDRESS
village. -X- _ O

**Comment by polm** [comment, marked as answer]
It looks like you should just be able to use [`spacy convert`](https://spacy.io/api/cli#convert) with the `conll` format. Did you try that?
